### PR TITLE
Add f-droid badge to README.md

### PR DESCRIPTION
**Pull request by RomainL972**
Now that the app has been added to F-droid, I think a badge could be added in the README to let users know they can download the app from there.

### CODE REVIEW

**Comment by RomainL972** [comment]
PS : Is it possible to tag a new release so all the latest changes can make it into a new F-droid release ?

File: README.md [hunk 1]
```diff
@@ -1,6 +1,9 @@
 # e-Passport NFC Reader
 
 [![Get it on Google Play](http://www.tananaev.com/badges/google-play.svg)](https://play.google.com/store/apps/details?id=com.tananaev.passportreader)
+[<img src="fdroid-badge.png"
+    alt="Get it on F-Droid"
+    height="40">](https://f-droid.org/packages/com.tananaev.passportreader)
```
Comment (tananaev) [inline README.md:6]:
```suggestion
[![Get it on Google Play](http://www.tananaev.com/badges/google-play.svg)](https://play.google.com/store/apps/details?id=com.tananaev.passportreader) [![Get it on F-Droid](http://www.tananaev.com/badges/f-droid.svg)](https://f-droid.org/packages/com.tananaev.passportreader)
```

Comment (RomainL972) [reply README.md]: Done

**Comment by tananaev** [comment]
Merged, thanks.

**Pull request merged**